docs/components/hand-controls.md

```diff
@@ -29,16 +30,20 @@ named, and handles hand animations.
 
 ## Events
 
-| Event Name | Description                                          |
-| ---------- | -----------                                          |
-| gripclose  | Grip buttons pressed. Hand is closed.                |
-| gripopen   | Grip buttons released. Hand is open.                 |
-| pointup    | Trigger button pressed. Index finger pointing up.    |
-| pointdown  | Trigger button released. Index finger pointing down. |
-| thumbup    | Thumbpad pressed. Thumb is pointing up.              |
-| thumbdown  | Thumbpad released. Thumb is pointing down.           |
+| Event Name    | Description                                                        |
+| ----------    | -----------                                                        |
+| gripclose     | 'fist': grip active, trigger active, trackpad/surface active       |
```

Comment:
I don't find these descriptions very useful. We should just describe the gesture in normal prose. e.g:
'The hand closes into a fist`